tasks/main.yml

```diff
@@ -23,7 +23,7 @@
   when: pip_installed.stat.exists == false
 
 - name: Upgrade pip
-  pip: name=pip state=latest executable="{{python_pip_path}}"
+  pip: name=pip state=latest executable="{{python_pip_path}}" umask=0002
```

Comment:
This should be a variable in _defaults/main.yml_, something like `python_pip_umask`, and have a comment explaining it.